### PR TITLE
Typing page

### PR DESCRIPTION
**Pull request by ljcrafty**
- Adding ms to timer component and finish screen
- Adding error indicator for errors on a space character

### CODE REVIEW

File: src/TypingPage/TypingPage.tsx [hunk 1]
```diff
@@ -27,10 +28,19 @@ class TypingPage extends React.Component<TypingProps, TypingState> {
...
+        for(let i = 0; i < insults.length; i++) {
+            count += insults[i].length;
+        }
+
         this.state = {
             currentInsult: 0,
             typedText: "",
-            isFinished: false
+            isFinished: false,
+            numChars: count
```
Comment (gldias) [inline src/TypingPage/TypingPage.tsx]: Is this used anywhere? Or was it part of the "double work" for calculating the typing speed?

Comment (ljcrafty) [reply src/TypingPage/TypingPage.tsx]: Good catch!

**Comment by Alan19** [comment]
Does this take into account my changes for calculating the typing speed in the finish modal?

**Comment by ljcrafty** [comment]
> Does this take into account my changes for calculating the typing speed in the finish modal?

it will when I merge master in again

**Pull request merged**